src/EmbedExtensionFactory.php

```diff
@@ -29,6 +29,8 @@ class EmbedExtensionFactory {
 		'ExternalContentFileExtensionWhitelist' => [ 'md' ],
 		'wgExternalContentEnableEmbedFunction' => true,
 		'wgExternalContentEnableBitbucketFunction' => true,
+		'ExternalContentDefaultExpiry' => 86400,
```

Comment:
should these be prefixed with 'wg'?

Reply:
With wg

I made it consistent in https://github.com/ProfessionalWiki/ExternalContent/pull/18